helm-service/deploy/service.yaml

```diff
@@ -92,7 +92,7 @@ spec:
           - name: PUBSUB_URL
             value: 'nats://keptn-nats-cluster'
           - name: PUBSUB_TOPIC
-            value: 'sh.keptn.event.deployment.triggered,sh.keptn.event.release.triggered,sh.keptn.event.action.triggered,sh.keptn.event.service.delete.finished'
+            value: 'sh.keptn.event.deployment.triggered,sh.keptn.event.release.triggered,sh.keptn.event.action.triggered,sh.keptn.event.service.delete.finished,sh.keptn.event.rollback.triggered'
```

Comment:
please also adapt the subscriptions in the installer helm chart: https://github.com/keptn/keptn/blob/6395ec1283db9f6479a66c9d6bbe47637eb32987/installer/manifests/keptn/charts/continuous-delivery/templates/continuous-deployment.yaml#L214